core/blockchain.go

```diff
@@ -68,15 +68,15 @@ const (
 	maxTimeFutureBlocks                = 30
 	badBlockLimit                      = 10
 	triesInMemory                      = 128
-	shardCacheLimit                    = 2
+	shardCacheLimit                    = 4
 	commitsCacheLimit                  = 10
 	epochCacheLimit                    = 10
 	randomnessCacheLimit               = 10
 	validatorCacheLimit                = 1024
 	validatorStatsCacheLimit           = 1024
 	validatorListCacheLimit            = 10
 	validatorListByDelegatorCacheLimit = 1024
-	pendingCrossLinksCacheLimit        = 2
+	pendingCrossLinksCacheLimit        = 10
```

Comment:
the shardCache and crossLinkCache only contains one key, increase cache size will not take any effect

Reply:
ok. sounds good.

Reply:
ShardCache is for shard state. I made it 10.